Empire/Empire.cpp

```diff
@@ -1024,8 +1040,8 @@ void ProductionQueue::Update() {
     auto available_pp = AvailablePP(industry_resource_pool);
     float pp_in_stockpile = industry_resource_pool->Stockpile();
     TraceLogger() << "========= pp_in_stockpile:     " << pp_in_stockpile << " ========";
-    const Meter* pp_use_limit = empire->GetMeter("METER_IMPERIAL_PP_USE_LIMIT");
-    float available_stockpile = pp_use_limit? std::min(pp_in_stockpile, pp_use_limit->Current()) : pp_in_stockpile;
+    float stockpile_limit = StockpileCapacity();
+    float available_stockpile = std::min(pp_in_stockpile, stockpile_limit);
```

Comment:
It seems like there are a few places, like here, and the next change below, where the previous reference to ```pp_use_limit->Current()``` has been changed to ```stockpile_limit```, but given the new meaning for the limit it seems to me the reference isn't needed here at all-- I didn't track down where the actual stockpile value is set, but I expect that you subject it to the limit at that time, and so here we perhaps don't need the new variable anymore, or at least could simply say
``` float available_stockpile = pp_in_stockpile```

Reply:
Here it needs to know what amount of stockpile PP is available (the lesser of what is in the stockpile and how much can be withdrawn per turn) when setting the spending on each item on the production queue, not just when calculating the new stockpile amount for the next turn based on the results of those allocations.